Automattic/vip-go-mu-plugins
# Validate only the active indexables instead of trying post and user no matter what (and failing to validate term and comment)

## Description

Our `validate-counts` command tried to count posts and users, but the users are not typically active, resulting in `Cannot find indexable 'user', probably the feature is not enabled`. This is not particularly helpful and could be anxiety-inducing.

Now we switch iterating over only active indexables, which would include Term and Comment too.


## Changelog Description

### Plugin Updated: Enterprise Search

We have updated `wp vip-search health validate-counts` command to only validate active Indexables.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Make sure User feature is not enabled.
1. Run the command.
1. Behold, no more user-related message.
